pmm-app/tests/pages/qanPage.js

```diff
@@ -52,6 +52,13 @@ module.exports = {
     tablesTabContents: "//div[@class='card-body']//pre",
     copyQueryButton: "//button[@id='copyQueryExample']",
     spinnerLocator: "//i[@class='fa fa-spinner fa-spin spinner']",
+    newQANPanelContent: '.panel-content',
+    countOfItems: "//*[@id='query-analytics-data']/div/div[2]/div/div[1]/div/div[2]/div/span",
```

Comment:
@IvanaWzientkova I don't think it would be good to use * in locator, please request you to change it to a specific element